src/main/java/org/openjdk/jextract/impl/DeclarationImpl.java

```diff
@@ -345,13 +341,91 @@ public int hashCode() {
         }
     }
 
+    // attributes
```

Comment:
Now all attributes, and relevant wither/getter/predicates are all defined here.

Reply:
Nice!